addons/mrp/migrations/10.0.2.0/pre-migration.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent <http://www.eficent.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+from odoo import tools
+
+_column_copies = {
+    'mrp_production': [
+        ('state', None, None)
+    ],
+    'mrp_workorder': [
+        ('state', None, None)
+    ],
+}
+
+_column_renames = {
+    'stock_move': [
+        ('consumed_for', None),
+    ],
+}
+
+_field_renames = [
+    ('mrp.bom', 'mrp_bom', 'product_uom', 'product_uom_id'),
+    ('mrp.bom.line', 'mrp_bom_line', 'product_uom', 'product_uom_id'),
+    ('mrp.production', 'mrp_production', 'date_planned', 'date_planned_start'),
+    ('mrp.production', 'mrp_production', 'move_lines', 'move_raw_ids'),
+    ('mrp.production', 'mrp_production', 'product_uom', 'product_uom_id'),
+    ('mrp.production', 'mrp_production', 'workcenter_lines', 'workorder_ids'),
+    ('mrp.routing', 'mrp_routing', 'workcenter_lines', 'operation_ids'),
+    ('mrp.workcenter', 'mrp_workcenter', 'capacity_per_cycle', 'capacity'),
+    ('mrp.workorder', 'mrp_workorder', 'date_planned', 'date_planned_start'),
+]
+
+_table_renames = [
```

Comment:
Use also `model_renames` method

Reply:
how? I proved putting it before and after the table_renames and in both cases fails